shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -163,10 +163,13 @@ - (instancetype)initWithSemanticsObject:(SemanticsObject*)semanticsObject {
   return self;
 }
 
+- (void)willRemoveFromAccessibilityBridge {
+  [self removeFromSuperview];
+}
+
```

Comment:
`removeFromSuperview` is called in the dealloc of the `FlutterScrollableSemanticsObject`, are these objects leaking, that's why they aren't getting removed from the superview?  If so we should fix the leak instead.

Reply:
it seems like the superview will hold a reference on this object. The object was never dealloc, so it just keeps adding more UIScrollable :D

Reply:
Makes sense, I think we should investigate what I've suggested in my last review before accepting this change.